doc/tutorials/acrn-sanity-test.rst

```diff
@@ -0,0 +1,318 @@
+
+ST_ACRNIO_SOS_Storage
+*********************
+
+* Description
+
+This test case focus on checking if eMMC works correctly in Service OS.
+
+* Prerequisites
+
+eMMC is installed
+
+* Test Steps
+
+1. Boot into Service VM
+
+   .. code-block:: none
+  
+   $ fdisk -l 
+
+#. Find eMMC block and mount it to read/write
+   NVME disk in /dev/nvmen0p1
+   SATA disk in /dev/sda3
+
+* Expected Result
+
+eMMC can be used normally in Service VM
+
+Graphics Virtualization (GVT)
+*****************************
+
+ST_ACRNGT_SOS_Rendering_Weston_APP
+**********************************
+
+* AcrnGT provide virtual display on User VM: run weston /weston-simple-egl in User VM,
+
+workload will show normally on User VM
+
+* Prerequisites
+
+System boot up normally
+
+Install ias weston related bundle
+
+   .. code-block:: none
+
+      $ sudo swupd bundle-add software-defined-cockpit x11-server 
+
+   .. note::
+    
+      do not install bundle desktop, these two may had conflict
+
+* Test Steps
+
+1. Boot up hypervisor and Service VM
+#. Start ias weston by command in Service VM & User VM :
+#. cd /usr/share/ias/examples/
+#. run ./weston-simple-shm in Service VM
+
+   .. code-block:: none
+
+     $ sudo systemctl enable ias
+     $ sudo systemctl start ias
+     $ sudo export XDG_RUNTIME_DIR=/run/ias
+     $ sudo weston-simple-egl
+
+* Expected Result
+
+Able to run test apps in Service VM without exception and display normally
+
+TC_ACRNGT_SOS_LaaG_Rendering_WestonAPP
+**************************************
+
+* Description
+
+AcrnGT support GPU sharing feature, run weston apps weston-simple-shm in Service VM and User VM,
+both Service VM and User VM will show workload normally
+
+* Prerequisites
+
+System boot up normally
+Install ias weston related bundle
+
+   .. code-block:: none
+
+      $ sudo swupd bundle-add software-defined-cockpit x11-server 
+
+   .. note::
+
+      do not install bundle desktop, these two may had conflict
+
+* Test Steps
+
+1. Boot up hypervisor, Service VM and User VM
+#. Start ias weston by command in Service VM & User VM:
+#. cd /usr/share/ias/examples/
+#. Run ./weston-simple-egl in User VM
+
+   .. code-block:: none
+
+      $ sudo systemctl enable ias
+      $ sudo systemctl start ias
+      $ sudo export XDG_RUNTIME_DIR=/run/ias
+      $ sudo weston-simple-egl
+      $ sudo cd /usr/share/ias/examples/
+      $ sudo ./weston-simple-shm
+
+* Expected Result
+
+Able to run test apps in Service VM and User VM synchronously
+
+ST_ACRNGT_LaaG_Rendering_Weston_OpenGL
+**************************************
+
+* Description
+
+AcrnGT provide virtual display on User VM: run weston /weston-simple-egl in User VM,
+workload will show normally on User VM
+
+* Prerequisites
+
+System boot up normally Install ias weston related bundle
+
+   .. code-block:: none
+
+      $ sudo swupd bundle-add software-defined-cockpit x11-server 
+
+   .. note:: 
+
+      do not install bundle desktop, these two may had conflict
+
+* Test Steps
+
```

Comment:
Same comments as for the similar paragraph above.

Reply:
I update 3 GVT case about weston . One is to start Westen in Service OS. One is to start One is to start Weson in User OS.Another one is to start Weson both in User OS and  Service OS.
Run weston-simple-egl and weston-simple-shm will show a different picture.